gm4_ziprails/data/gm4_ziprails/functions/checks/cable.mcfunction

```diff
@@ -0,0 +1,2 @@
+execute if block ~ ~1 ~ #gm4_ziprails:cable[attached=true] run function gm4_ziprails:ziprail/zipping
```

Comment:
Please add a heading comment for this function. Also I find it a bit strange to create a folder for this one function. Especially since "checks" is not very descriptive. Most functions check something and then do something with the result.